Lib/test/test_logging.py

```diff
@@ -63,9 +64,6 @@
 from socketserver import (ThreadingUDPServer, DatagramRequestHandler,
                           ThreadingTCPServer, StreamRequestHandler)
 
-with warnings.catch_warnings():
```

Comment:
The change removes `with warnings.catch_warnings():` but apparently, it's useless. The test still pass without it:

```
$ ./python -Werror -m test test_logging test_smtplib -j0
Using random seed: 1598323201
0:00:00 load avg: 0.96 Run 2 tests in parallel using 2 worker processes
0:00:02 load avg: 1.37 [1/2] test_smtplib passed
0:00:18 load avg: 1.22 [2/2] test_logging passed

== Tests result: SUCCESS ==

All 2 tests OK.

Total duration: 18.9 sec
Total tests: run=317 skipped=2
Total test files: run=2/2
Result: SUCCESS
```